infraohjelmointi_api/tests/test_Projects.py

```diff
@@ -2733,49 +2733,54 @@ def test_date_validation_on_lock(self):
         )
 
     def test_class_location_validation(self):
+        ETELAINEN_SUURPIIRI = "Eteläinen suurpiiri"
+        LANTINEN_SUURPIIRI = "Läntinen suurpiiri"
+        KESKINEN_SUURPIIRI = "Keskinen suurpiiri"
+        OSTERSUNDOMIN_SUURPIIRI = "Östersundomin suurpiiri"
+
         district_1 = ProjectLocation.objects.create(
             id=self.projectDistrict_4_Id,
-            name="Eteläinen",
+            name=ETELAINEN_SUURPIIRI,
             parent=None,
-            path="Eteläinen",
+            path=ETELAINEN_SUURPIIRI,
         )
         district_2 = ProjectLocation.objects.create(
             id=self.projectDistrict_5_Id,
-            name="Läntinen",
+            name=LANTINEN_SUURPIIRI,
             parent=None,
-            path="Läntinen",
+            path=LANTINEN_SUURPIIRI,
         )
         district_3 = ProjectLocation.objects.create(
             id=self.projectDistrict_6_Id,
-            name="Keskinen",
+            name=KESKINEN_SUURPIIRI,
             parent=None,
-            path="Keskinen",
+            path=KESKINEN_SUURPIIRI,
         )
         district_4 = ProjectLocation.objects.create(
             id=self.projectDistrict_7_Id,
-            name="Östersundom",
+            name=OSTERSUNDOMIN_SUURPIIRI,
             parent=None,
-            path="Östersundom",
+            path=OSTERSUNDOMIN_SUURPIIRI,
         )
         district_1.childLocation.create(
             id=self.projectDivision_3_Id,
             name="Munkkiniemi",
-            path="Eteläinen/Munkkiniemi",
+            path="Eteläinen suurpiiri/Munkkiniemi",
         )
         district_2.childLocation.create(
             id=self.projectDivision_4_Id,
             name="Munkkiniemi",
-            path="Läntinen/Munkkiniemi",
+            path="Läntinen suurpiiri/Munkkiniemi",
         )
         district_3.childLocation.create(
             id=self.projectDivision_5_Id,
             name="Munkkiniemi",
-            path="Keskinen/Munkkiniemi",
+            path="Keskinen suurpiiri/Munkkiniemi",
         )
         district_4.childLocation.create(
             id=self.projectDivision_6_Id,
             name="ostersundomTest",
-            path="Östersundom/ostersundomTest",
+            path="Östersundomin suurpiiri/ostersundomTest",
```

Comment:
These hard coded district texts (e.g. "Eteläinen suurpiiri" in the row 2768) could be replaced with the variables also `path="{ETELAINEN_SUURPIIRI}/Munkkiniemi",`?